src/evotorch/algorithms/functional/funcdcem.py

```
@@ -0,0 +1,470 @@
+        device: If given as a string or as a `torch.device` instance, the
+            evolutionary search will be performed on this specified device.
+    """
+    objective_sense = str(objective_sense)
+    if objective_sense == "min":
+        maximize = False
+    elif objective_sense == "max":
+        maximize = True
+    else:
+        raise ValueError(
+            "`objective_sense` was expected as 'min' or 'max', but it was received as: {repr(objective_sense)}."
+        )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)